backend/openapi.yml

```diff
@@ -1076,29 +1076,69 @@ paths:
                         "purpose": ""
                       }
                     ]
-  /api/v1/can-funding-summary/{can_id}:
+  /api/v1/can-funding-summary:
     get:
       tags:
         - CAN Funding Summary
-      operationId: getFundingTotalsByCanId
-      description: Get Funding Totals by CAN Id
+      operationId: getFundingTotals
+      description: Get Funding Totals
       parameters:
         - $ref: "#/components/parameters/simulatedError"
+        - name: can_ids
+          in: query
+          description: Comma-separated list of CAN Ids (required)
+          required: true
+          schema:
+            type: array
+            items:
+              type: integer
+            example: [ 12345, 12346, 12347 ]
         - name: fiscal_year
           in: query
+          description: Fiscal year (optional)
           schema:
             type: string
-          example: "2023"
-        - in: path
-          name: can_id
-          description: CAN Id
-          required: true
+            example: "2023"
+        - name: active_period
+          in: query
+          description: List of active periods (optional)
           schema:
-            type: integer
-            format: int32
-            minimum: 0
-            default: 0
-      responses:
+            type: array
+            items:
+              type: integer
+            example: [ 1, 5 ]
+        - name: transfer
+          in: query
+          description: List of CAN Method of Transfer (optional)
+          schema:
+            type: array
+            items:
+              type: string
+              enum:
+                - DIRECT
+                - COST_SHARE
+                - IAA
+                - IDDA
+            example: [ "DIRECT", "IAA" ]
+        - name: portfolio
+          in: query
+          description: List of portfolios (optional)
+          schema:
+            type: array
+            items:
+              type: string
```

Comment:
I'm unsure if this should be the portfolio ID, name, or abbreviation. 🤔 Thoughts?

Reply:
think the `abbreviation` or `id` would be more slug-freindly:
<img width="379" alt="image" src="https://github.com/user-attachments/assets/86344c1e-43b0-4340-b15d-740f5fbd86e6">